### ISSUE TITLE
What happened to parts 4 and 6?

### ISSUE THREAD

**Issue by bwims** [SuperHouse/AQS#9]
Hi, what happened to:

Part 4: Recording and reporting Air Quality Sensor data (coming soon)
Part 5: DIY Air Quality Sensor “Pro” version (coming soon)

Coming soon?  It's been 3 years!  :-)

**Comment by bwims** [comment]
Bump!

**Comment by austinscreations** [comment]
this has been discussed a few times in various livestreams over the years.

1) backend stuff has changed quite a bit over the years that would require a full re-shoot of content to make it useful / relevant
2) life events have happened
3) professional paid work takes precedence. and jons mostly a one man shop with occasional help from his son.

**Comment by bwims** [comment]
Fair enough.  In that case, how much effort would there be in coming clean and editing the appropriate information into the blog entries?  I.E. don't raise expectations unnecessarily.  Say.  "Nope.  Forget about any future development. "

Thankfully, ESPhome caters for this device and provides all the extras required.
